src/supernode/PosSaleObject.cpp

```diff
@@ -133,27 +133,30 @@ void supernode::PosSaleObject::ContinueInit() {
 	inbr.SenderIP = m_DAPIServer->IP();
 	inbr.SenderPort = m_DAPIServer->Port();
 	if( !m_SubNetBroadcast.Send(dapi_call::PosProxySale, inbr, outv) || outv.empty() ) {
-		LOG_PRINT_L5("!Send dapi_call::PosProxySale");
+        LOG_PRINT_L0("!Send dapi_call::PosProxySale");
```

Comment:
Maybe it's better to log errors to LOG_ERROR?

Reply:
Fixed